Winget-AutoUpdate/functions/Test-Network.ps1

```diff
@@ -1,55 +1,58 @@
 #Function to check connectivity
 
 function Test-Network {
-    
+
     #Init
     $timeout = 0
 
+    # Workaround for ARM64 (Access Denied / Win32 internal Server error)
+    $ProgressPreference = 'SilentlyContinue'
```

Comment:
This will suppress the progress of all `Invoke-WebRequest` only.
It will not change anything else.